src/tetris_game.h

```diff
@@ -103,6 +112,13 @@ int rotate_block(int clockwise, struct game_contents *game_contents);
  */
 int new_game(struct game_contents **game_contents);
 
+/*
+ * Starts a game and takes a double pointer for game contents data.
+ * Also takes a seed to base the number generation from.
+ * @return 0
+ */
+int new_seeded_game(struct game_contents **game_contents);
```

Comment:
It seems like this function signature doesn't match the one present in `src/tetris_game.c`

Reply:
Do not know how this got through compilation for me..

Reply:
You have a better compiler than I do :smile: